Tests/FoundationEssentialsTests/GregorianCalendarTests.swift

```diff
@@ -3257,10 +3257,10 @@ final class GregorianCalendarTests : XCTestCase {
         XCTAssertEqual(cal.range(of: .dayOfYear, in: .year, for: leapYearDate), 1..<367)
 
         // Addition
-        let d1 = cal.add(.dayOfYear, to: date, amount: 1, inTimeZone: .gmt)
+        let d1 = try? cal.add(.dayOfYear, to: date, amount: 1, inTimeZone: .gmt)
```

Comment:
If we don't expect this to throw, could we mark the test as `throws` and just do `try` here? That'd mean that if an error is thrown, the test would fail and the failure would indicate the error information rather than just a `nil` value for `d1` which would probably help with debugging